CHANGELOG.asciidoc

```diff
@@ -47,6 +47,8 @@ https://github.com/elastic/beats/compare/v6.0.0-beta2...master[Check the HEAD di
 - Refactor the usage of prospector to input in the YAML reference and
   the system test. {pull}6121[6121]
 - Add IIS module to parse access log and error log. {pull}6127[6127]
+- Add validation for Stdin, when Filebeat is configured with Stdin and any other inputs, Filebeat
```

Comment:
I wonder if this is a bug fix or a breaking change :-)

Reply:
Its kind of both 🍁